backend/app/views/spree/admin/products/index.html.erb

```diff
@@ -25,19 +25,26 @@
           </div>
         </div>
         <div class="col-12">
-          <div class="field checkbox mb-2">
-            <label>
-              <%= f.check_box :deleted_at_null, {checked: params[:q][:deleted_at_null] == '0'}, '0', '1' %>
-              <%= Spree.t(:show_deleted) %>
-            </label>
-          </div>
+          <%= f.field_container :all_stores do %>
```

Comment:
Add the ability to **Show Products For All Stores** in the Products Filter, a power feature for Spree when managing products.

Additionally, replace checkboxes with Bootstrap Custom Checkboxes for standard look and feel across browsers, and they are now the standard in BS5.